src/ui/public/utils/lodash-mixins/string.js

```diff
@@ -10,7 +10,7 @@ export default function (_) {
      * @return {string}
      */
     asPrettyString: function (val) {
-      if (val === null || val === undefined) return ' - ';
+      if (val === null || val === undefined) return '(empty)';
```

Comment:
I decided to revert this change. I think its out of scope for this particular issue and can be handled in a separate PR. Regardless of whether we use ` - ` or `(empty)` we'll want to style it differently to indicate that its not the actual value of the field.